client/transport/src/ws/mod.rs

```diff
@@ -195,6 +197,20 @@ impl TransportSenderT for Sender {
 		Ok(())
 	}
 
+	/// Sends out a ping request. Returns a `Future` that finishes when the request has been
+	/// successfully sent.
```

Comment:
Since we've been talking about cancel safety a bit, and not really relaetd to this PR, but do you reckon it is worth adding an issue to add docs to functions to note when they aren't cancel safe @niklasad1?

Reply:
sounds like a good idea.